src/libs/TransactionUtils.js

```diff
@@ -389,6 +389,16 @@ function getValidWaypoints(waypoints, reArrangeIndexes = false) {
     return validWaypoints;
 }
 
+/**
+ * Returns the most recent transactions in an object
+ * @param {Object} transactions - Shaped with transctions as keys and inserted dated as values
+ * @param {Number} total
+ * @returns {Array}
+ */
+function getLatestTransactions(transactions, total = 2) {
```

Comment:
Comment explains `most recent`, why not name the method `getRecentTransactions`?